ci-operator/config/openshift/elasticsearch-operator/openshift-elasticsearch-operator-master.yaml

```diff
@@ -93,13 +93,13 @@ operator:
   - pullspec: quay.io/openshift-logging/curator5
     with: pipeline:logging-curator5
 promotion:
-  name: "5.7"
+  name: "5.8"
   namespace: logging
 releases:
   latest:
     release:
       channel: fast
-      version: "4.12"
+      version: "4.13"
```

Comment:
If I'm interpreting the errors in the rehearsals correctly, then there's no "fast-4.13" yet. "candidate-4.13" seems to work, if I try out the API myself, so maybe we need to change the channel until 4.13 is released.